opal/mca/base/mca_base_var.c

```diff
@@ -1543,12 +1543,12 @@ int mca_base_var_register(const char *project_name, const char *framework_name,
     }
 
     OPAL_LIST_FOREACH_DECL(alias_item, &alias->component_aliases, mca_base_alias_item_t) {
-        mca_base_var_syn_flag_t flags_tmp = 0;
+        mca_base_var_syn_flag_t flags_tmp = flags;
         if (alias_item->alias_flags & MCA_BASE_ALIAS_FLAG_DEPRECATED) {
             flags_tmp = MCA_BASE_VAR_SYN_FLAG_DEPRECATED;
         }
         (void) mca_base_var_register_synonym(ret, project_name, framework_name,
-                                             alias_item->component_alias, variable_name, flags);
+                                             alias_item->component_alias, variable_name, flags_tmp);
```

Comment:
What's the point ? The arts are read only, you can safely pass directly flags.

Reply:
I was just fixing what I think was the original intent of the code (i.e., conditionally also pass in `MCA_BASE_VAR_SYNC_FLAG_DEPRECATED`).  I don't have much of an opinion on whether to use `flags_tmp` or the original `flags`.  I've asked @hjelmn for his opinion on this as well.

Reply:
SGTM